src/components/gridList/gridList.js

```diff
@@ -375,7 +412,8 @@ function GridLayoutFactory($mdUtil) {
       reflow: function(customAnimatorFn) {
         reflowTime = $mdUtil.time(function() {
           var animator = customAnimatorFn || defaultAnimator;
-          tiles.forEach(function(it) {
+          animator(gridStyles.grid.element, gridStyles.grid.style);
```

Comment:
I thought it might be interesting to have that flexibility, and even more so when I do what you're suggesting below. Maybe you want to have the grid flip over and reveal the new layout or something. :)